ui/.env.sample

```diff
@@ -1,9 +1,9 @@
-VITE_API_URL=
-VITE_API_USERNAME=
-VITE_API_PASSWORD=
+VITE_API_URL=http://localhost:3002
+VITE_API_USERNAME=user-ui
+VITE_API_PASSWORD=password-ui
 
-VITE_ISSUER_DID=
-VITE_ISSUER_NAME=
-VITE_ISSUER_LOGO=
+VITE_ISSUER_DID=<YOUR_ISSUER_API_UI_ISSUER_DID_FROM_ENVAPI>
+VITE_ISSUER_NAME=<YOUR_ISSUER_API_UI_ISSUER_NAME_FROM_ENVAPI>
+VITE_ISSUER_LOGO=<YOUR_ISSUER_API_UI_ISSUER_LOGO_FROM_ENVAPI>
 
-VITE_BLOCK_EXPLORER_URL=
+VITE_BLOCK_EXPLORER_URL=https://mumbai.polygonscan.com
```

Comment:
We definitely don't want to add anything here! This is because the UI can't validate if these are correct values, only if they are present - so the logo being a random placeholder will result in an unhandled error, for example. We need the user to consciously copy the values from the other `.env` files as instructed in the README. NB that these values are only needed if the user is running the UI in HMR (local development) mode.

Reply:
I will add a comment to the sample file to clarify this.